Moda.Web/src/moda.web.reactclient/src/app/planning/roadmaps/components/roadmaps-timeline.tsx

```diff
@@ -50,18 +45,23 @@ const GroupTemplate = (props: GroupTemplateProps<RoadmapChildrenDto>) => {
 }
 
 const getDataGroups = (
```

Comment:
@destacey - I created this PR as a draft because of this area. My brain has been extra mushy today, and I need to take another look at this when my brain hopefully isn't mush anymore + I want to talk to you about what this is doing and how I believe the `groups[]` array type depends on whether or not `groupItems` gets passed in.

Reply:
This looks great thank you.  I've added a few comments/questions.